src/python_testing/TC_TSTAT_4_2.py

```diff
@@ -61,32 +61,36 @@ async def write_presets(self, endpoint, presets) -> Status:
         result = await self.default_controller.WriteAttribute(self.dut_node_id, [(endpoint, cluster.Attributes.Presets(presets))])
         return result[0].Status
 
-    async def send_edit_preset_request_command(self,
+    async def send_edit_atomic_request_begin_command(self,
                                                endpoint: int = None,
                                                expected_status: Status = Status.Success):
         try:
-            await self.send_single_cmd(cmd=cluster.Commands.StartPresetsSchedulesEditRequest(timeoutSeconds=180),
+            await self.send_single_cmd(cmd=cluster.Commands.AtomicRequest(requestType=0, 
```

Comment:
Followup (no need to do it in this PR): Can we use the right named enum value for that requestType?  And similar elsewhere.

Reply:
Filed https://github.com/project-chip/connectedhomeip/issues/34742